src/test/librbd/test_Migration.cc

```diff
@@ -273,6 +273,7 @@ struct TestMigration : public TestFixture {
   void flush() {
     ASSERT_EQ(0, api::Io<>::flush(*m_ref_ictx));
     ASSERT_EQ(0, api::Io<>::flush(*m_ictx));
+    usleep(100000);
```

Comment:
Nit: ??

Reply:
Here we need to flush data to OSDs.

Reply:
We cannot expect sleeps to work 100% of the time -- if you need data to flush to the OSDs, you need a way to force it w/o sleeping.

Reply:
This sleep has been removed.